### ISSUE TITLE
Update Gradient Descent Fixed Step Size

### ISSUE THREAD

**Issue by vp314** [numoptim/OptimizationMethods.jl#10]
# Objective

For the Gradient Descent with Fixed Step Size method, we need to update the naming of the gradient norm history from `gra_val_hist` to `grad_val_hist` to keep the naming consistent with the API from `NLPModels`

# Summary of Changes
- [ ] Update Gradient Descent with Fixed Step Size variable name from `gra_val_hist` to `grad_val_hist`
- [ ] Update docstring to reflect the change in naming
- [ ] Update the examples to reflect the change in naming
- [ ] Update the tests to reflect the change in naming
